src/ext/oblivc/obliv_bits.c

```diff
@@ -26,6 +26,15 @@ static __thread ProtocolDesc *currentProto;
 static inline bool known(const OblivBit* o) { return !o->unknown; }
 
 // --------------------------- Transports -----------------------------------
+
+static bool transportProfilingEnabled = false;
```

Comment:
Globals always have weird race condition issues. I know it seems like it shouldn't be a problem here, but then it always bites back when you least expect. Plus it is less granular. I can think of two alternatives that I like better:

1) Move this variable inside ProtocolDesc, so as to have only one ugly global pointer to struct to ever deal with.
2) Even better, have people call different functions. E.g. instead of protocolAcceptTcp2P(), they call protocolAcceptProfiledTcp2().

In either case, we have to worry about copying these fields when copying structs. In option 2, there is an easy choke point where we can do this for all copying, split(). Might be a little harder for ProtocolDesc. What do you think?